### ISSUE TITLE
Finetuning process

### ISSUE THREAD

**Issue by francislata** [jik876/hifi-gan#71]
@jik876 -  I would like to ask about your finetuning process. Specifically, what are some of the things to consider?

Let's say I'm training from the LJSpeech model to another single speaker model.

1) Do I consider using the learning rate scheduler at all, or just use a constant learning rate for the generator and discriminator?

2) For the learning rate themselves (and I guess this really depends on the dataset too), do you use a much lower learning rate?

3) Do you only train the generator, the discriminator, or both, when fine-tuning?

**Comment by jik876** [comment]
Thanks for your interest.
1. In our fine-tuning experiments, the effect of the scheduling may have been relatively small, but if you use a different speaker dataset from the original and adjust the initial learning rate, using the learning rate scheduling can lead to better quality.
2. We used a learning rate calculated by the scheduler according to training step. You are using a different speaker dataset from the original, so this may need to be adjusted.
3. We trained both.
